Tasks/ExtractFiles/package.json

```diff
@@ -0,0 +1,19 @@
+{
+  "name": "extractFiles",
+  "version": "1.0.0",
+  "description": "Extract Files Task",
+  "main": "extractFiles.js",
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/Microsoft/vsts-tasks.git"
```

Comment:
Actually, package.json is only for extensions and shouldn't be included with this task.